.golangci.yml

```diff
@@ -5,6 +5,7 @@ linters:
   disable-all: true
   enable:
   - golint
+  - gosimple
```

Comment:
Noticed you opened a few of these lately. I think its amazing! 👏  But my suggestion is it might be amazing to also open issues for other external folks to do, as its really great first issue. What do you think?

Reply:
It happens that I had all the changes already prepared on my machine and the last one would be for the errcheck linter.